std_msgs/Header header
std_msgs/String name
geometry_msgs/Pose2D point
geometry_msgs/Point centriod
sensor_msgs/Image image
